### PR TITLE
Do not report issues in Java only projects

### PR DESCRIPTION
**Pull request by tgodzik**
Related to https://github.com/scalacenter/bloop/issues/880

### CODE REVIEW

File: metals/src/main/scala/scala/meta/internal/metals/Doctor.scala [hunk 1]
```diff
@@ -254,13 +253,18 @@ final class Doctor(
...
-        if (isSemanticdbEnabled) {
-          Icons.unicode.check
-        } else {
+        if (isSemanticdbNeeded) {
           Icons.unicode.alert
+        } else {
+          Icons.unicode.check
         }
       val center = "style='text-align: center'"
+      val fixRecommendation =
```
Comment (marek1840) [inline metals/src/main/scala/scala/meta/internal/metals/Doctor.scala]: Why is it called **fix**Recommendation?

File: metals/src/main/scala/scala/meta/internal/metals/Doctor.scala [hunk 2]
```diff
@@ -269,7 +273,9 @@ final class Doctor(
           .element("td", center)(_.text(completions))
           .element("td", center)(_.text(references))
           .element("td")(
-            _.raw(recommendation(scalaVersion, isSemanticdbEnabled, target))
+            _.raw(
```
Comment (marek1840) [inline metals/src/main/scala/scala/meta/internal/metals/Doctor.scala]: Can it be put in a single line?

File: metals/src/main/scala/scala/meta/internal/metals/Doctor.scala (hunk 1 above)
Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/metals/Doctor.scala]: Recommendation about the fix

Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/metals/Doctor.scala]: Changed to recommenedFix, ok?

**Pull request merged**